src/session.rs

```diff
@@ -1020,7 +1019,10 @@ pub struct PacketReceipt {
 
 /// Find the PayloadParams for the given Pt, either when the Pt is the main Pt for the Codec or
 /// when it's the RTX Pt.
-fn main_payload_params(c: &CodecConfig, pt: Pt) -> Option<&PayloadParams> {
+///
+/// Returns the `PayloadParams` for the main RTP stream and a `bool` indicating if the supplied pt
+/// was a repair pt.
+fn main_payload_params(c: &CodecConfig, pt: Pt) -> Option<(&PayloadParams, bool)> {
     c.iter()
-        .find_map(|p| (p.pt == pt || p.resend == Some(pt)).then_some(p))
+        .find_map(|p| (p.pt == pt || p.resend == Some(pt)).then(|| (p, p.pt != pt)))
```

Comment:
This seems like a caller concern. No need to alter the return signature, just compare the returned value.

Reply:
Aight